tools/validators/instance_validator/tests/fake_instances/BAD/missing_link_in_source_entity.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this is a bad example since the virtual entity has a link to
+# return_water_temperature_sensor_200 on the physical entity; however
+# the physical entity doesn't have that point, it has
+# return_water_temperature_sensor
+VIRTUAL-ENTITY-GUID:
+  type: HVAC/CHWS_WDT
+  code: VIRTUAL-ENTITY
+  links:
+    PHYSICAL-ENTITY-GUID:
+      return_water_temperature_sensor: return_water_temperature_sensor_200
+      supply_water_temperature_sensor: supply_water_temperature_sensor
+      thermal_power_capacity: thermal_power_capacity
+      differential_pressure_specification: differential_pressure_specification
+
+PHYSICAL-ENTITY-GUID:
+  type: HVAC/CHWS_WDT
+  code: PHYSICAL-ENTITY
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something generic like 1234567890123456

Reply:
done